pybossa/cache/projects.py

```diff
@@ -70,8 +70,9 @@ def browse_tasks(project_id, args, filter_user_prefs=False, user_id=None):
         return total_count, tasks
 
     filters, filter_params = get_task_filters(args)
```

Comment:
we can modify the` filters` list returned by `filters, filter_params = get_task_filters(args)` to include the where clause that excludes previously worked tasks, so that we don't need to change the main sql query.

Reply:
Moved to task_browse_helpers.py in https://github.com/bloomberg/pybossa/pull/614/commits/c55dac993b6d446665d5cd6253c23e0a06ee7075